### DISCUSSION TITLE
Daily and monthly comparision

### DISCUSSION THREAD

**Discussion by ich92** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#35] [Q&A]
Hello,
I hope someone can help me.
I'm looking for a way to have the daily and monthly values. For every day of the year and every month.
Is there any way to Do such a Thing in homeassistant with this plugin?

Thanks for your help.

**Comment by andrewoe** [comment]
Hi,
statistic card is the answer... 


![Bildschirmfoto 2024-04-23 um 09 46 01](https://github.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant/assets/75129561/aa85d91a-42aa-46fe-a608-1fe92694d12a)

**Reply by rubkec** [reply]
Do you got the YAML code for this?

**Comment by retoch44** [comment]
This works for me in configuraton.yaml:

utility_meter:
  solarertrag_daily:
    source: sensor.bkw_today_production
    cycle: daily
    unique_id: "Solarertrag täglich"
  solarertrag_weekly:
    source: sensor.bkw_today_production
    cycle: weekly
    unique_id: "Solarertrag wöchentlich"
  solarertrag_monthly:
    source: sensor.bkw_today_production
    cycle: monthly
    unique_id: "Solarertrag monatlich"
  solarertrag_yearly:
    source: sensor.bkw_today_production
    cycle: yearly
    unique_id: "Solarertrag jährlich"
   
Change source to your sensor name. You can add the new entities to your dashboards and it will also be available at your energy stats. Unique IDs are useful for changing output format in the ui.
